test/browser-test/generic/index.js

```diff
@@ -0,0 +1,34 @@
+const tf = require("@tensorflow/tfjs-core");
```

Comment:
Shall we use ES Modules?

Reply:
True, I'll fix that

Reply:
I just found out that Karma does not support ES6 import from installed packages, that means I cannot import an installed library in-browser test file using ES6 format. You can only import relative scripts. 

Ref https://stackoverflow.com/questions/57066106/typeerror-failed-to-resolve-module-specifier-open-wc-testing-relative-refer didn't help either. 

So we can stick to using require for browser testing unless there is a better solution

Reply:
How about using webpack `resolve.alias`? See https://webpack.js.org/configuration/resolve/.

Reply:
@yorkie This more of a Karma issue not webpack